Tests/Translator/Extractor/JMSTranslatorBundle/AdminExtractorTest.php

```diff
@@ -77,8 +77,21 @@ public function setUp()
 
         $logger = $this->getMock('Psr\Log\LoggerInterface');
 
-        $this->pool = new Pool($container, '', '');
-        $this->pool->setAdminServiceIds(array('foo_admin', 'bar_admin'));
+        $this->pool = $this->getMockBuilder('Sonata\AdminBundle\Admin\Pool')
+            ->disableOriginalConstructor()
+            ->getMock()
+        ;
```

Comment:
Aside from that it's good to go, I'll merge as soon as you fix this tiny problem.